core/src/main/java/tc/oc/pgm/Config.java

```diff
@@ -477,4 +477,15 @@ public static boolean isAppealVisible() {
       return getAppealMessage().length() > 0;
     }
   }
+
+  public static class Fireworks {
+
+    public static boolean postMatchEnabled() {
+      return getConfiguration().getBoolean("fireworks.post-match", false);
```

Comment:
Post match default is false? Why not true?

Reply:
It was suggested above by @Pablete1234 to have it default false. Personally I would have it as true. Want me to adjust it?